README.md

```diff
@@ -1,2 +1,288 @@
-# Balance On Wheels
-Modeling and Control of Two-Legged Wheeled Robot: In this project, we will be building a two wheeled robot (with a hip and knee joint) which can balance at variable heights including a provision for jumping.
+![](https://i.imgur.com/V6Vat3e.jpg)
+
+# <ins>Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Tech Stack](#tech-stack)
+  * [File Structure](#file-structure)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installations](#installations)
+  * [Execution](#execution)
+* [Theory and Approach](#theory-and-approach)
+* [Results and Demo](#results-and-demo)
+* [Future Work](#future-work)
+* [Troubleshooting](#troubleshooting)
+* [Contributors](#contributors)
+* [Acknowledgements and Resources](#acknowledgements-and-resources)
+* [License](#license)
+
+---
+
+# <ins>About the Project
+
+**Modeling and Control of Two-Legged Wheeled Robot: In this project, we will be building a two wheeled robot (with a hip and knee joint) which can balance at variable heights including a provision for jumping.**
+
+---
+
+## <ins>Tech Stack
+
+* <font color="red">C</font>
+* <font color="red">Python</font>
+* <font color="red">SolidWorks</font>
+* <font color="red">ESP-IDF</font>
+* <font color="red">AutoCad</font>
+* <font color="red">Creality</font>
+
+## <ins>File Structure
```

Comment:
This is not the correct file structure. I hope this is temporary

Reply:
It is temporary, once the final code is ready will update the file structure

Reply:
made changes!